elastic/apm-server
# package: Update `labels|data_stream` fields as ecs

## Motivation/summary

Updates the `elastic-package` binary to the latest version which allows
us to benefit from the latest changes. In particular, references to ecs
external fields may override some of the field properties.

This patch updates the `data_stream.*`, `@timestamp` and `labels` fields
to reference the external ecs definition.

## Checklist

~- [ ] Update [CHANGELOG.asciidoc](https://github.com/elastic/apm-server/blob/master/CHANGELOG.asciidoc)~
~- [ ] Documentation has been updated~

## How to test these changes

1. `make build-package`
2. `docker-compose up -d`
3. Install the APM Integration in Kibana.
4. Verify all the modified fields appear in the index template mappings with their defined ecs types.
5. Verify the "labels" field has `dynamic: true` set in all the index template mappings

## Related issues

Follow up from #6633
